bash_completions_test.go

```diff
@@ -74,26 +75,26 @@ func TestBashCompletions(t *testing.T) {
 		Run:                    emptyRun,
 	}
 	rootCmd.Flags().IntP("introot", "i", -1, "help message for flag introot")
-	rootCmd.MarkFlagRequired("introot")
+	er(rootCmd.MarkFlagRequired("introot"))
```

Comment:
For all instances of `er` could we use the exported function `Er` instead? 
Although it is part of the same package, I think we should strive to use our exported methods. What do you think?

Reply:
At first, I didn't want to export it, because it feels like an internal utility. However, then I had to export it (I don't remember why). Once it is exported, I guess it makes sense to remove the unexported version. I did that now.